src/content/reference/hi/p5/randomSeed.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: randomSeed
+module: गणित
+submodule: यादृच्छिक
+file: src/math/random.js
+description: >
+  <p> <a href='/reference/p5/random'>random()</a> और <a href='/reference/p5/randomGaussian'>randomGaussian()</a> 
+  के लिए बीज मान सेट करता है कार्य.</p>
+
+  <p>डिफ़ॉल्ट रूप से, <a href='/reference/p5/random'>random()</a> और <a href='/reference/p5/randomGaussian'>randomGaussian()</a> 
+  अलग-अलग परिणाम देते हैं हर बार एक स्केच चलाया जाता है। <code>randomSeed()</code> को निरंतर तर्क के साथ कॉल करने से, जैसे कि 
```

Comment:
Can this be rephrased? The grammar looks off

Reply:
done